spark/src/test/scala/org/apache/comet/exec/CometExecSuite.scala

```diff
@@ -218,6 +220,51 @@ class CometExecSuite extends CometTestBase {
     }
   }
 
+  test("test cast utf8 to boolean as compatible with Spark") {
```

Comment:
I think `CometExpressionSuite` is more proper place for the test?

Reply:
Thanks @viirya for quick review. Yes, makes sense, addressed.